### PR TITLE
Final mile procedures and updates

### LINKED ISSUE #2801: Friendly name cleanup
**Description:**

There are inconsistencies in the friendly names for related data elements. Some examples are below for TANF T6 record for the approved number of application data elements:

https://github.com/raft-tech/TANF-app/blob/ca6eecf9dada19b29ba5f35b3f77541856ad2ebb/tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py#L300

https://github.com/raft-tech/TANF-app/blob/ca6eecf9dada19b29ba5f35b3f77541856ad2ebb/tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py#L89

https://github.com/raft-tech/TANF-app/blob/ca6eecf9dada19b29ba5f35b3f77541856ad2ebb/tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py#L511

 

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_

- [ ] friendly names for [TANF T6 records](https://github.com/raft-tech/TANF-app/blob/develop/tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py) are consistent
- [ ] All friendly names in code base are crross-referenced with associated coding instructions and 1:1 match each item name
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [x] Miles & Victoria pair on Git processes and friendly name subject area
- [x] Replace friendly names in codebase with verbatim versions from coding instructions
- [x] Write truncated friendly names for wordy instruction edge cases
- [ ] Run Testing Checklist and confirm all tests pass

### PR DESCRIPTION
**Pull request by victoriaatraft**
## Summary of Changes
New PR to correct for some unwanted conflicts with develop 
Closes #2801

### CODE REVIEW

**Review by reitermb — APPROVED** [review]
Looks great to me! Thanks Victoria! And thank you Eric for the test fixes! 

Over to @ADPennington for official QASP approval

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m2.py]: @victoriaatraft @elipe17 why did we change this validator?

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py]: @elipe17 @victoriaatraft why this change?

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py]: @elipe17 @victoriaatraft same question as above.

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m5.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m5.py]: @elipe17 @victoriaatraft same questions as above.

Comment (victoriaatraft) [reply tdrs-backend/tdpservice/parsers/schema_defs/ssp/m5.py]: @ADPennington I will follow up with @EricYagerPersonalRepo on these right now. I know we had some conflicts that he helped us with.

**Comment by ADPennington** [comment]
adding blocked tag pending feedback on validator changes.

**Comment by victoriaatraft** [comment]
> adding blocked tag pending feedback on validator changes.

okay. I 100% deleted them, so sorry about that. @elipe17 will graciously add them back in so it can be good to go.

**Comment by ADPennington** [comment]
@victoriaatraft a few of these friendly names are still quite verbose. lets discuss options for a path forward at next UX sync. cc: @lfrohlich

File: tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t4.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t4.py]: this change needs to be prioritized for merging.

File: tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t5.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t5.py]: prioritize this change for merging

**Review by ADPennington — CHANGES_REQUESTED** [review]
@elipe17 @victoriaatraft i left comments on a couple of line changes that can be prioritized for  merging, perhaps on another branch. defer to dev on how best to go about that.

**Comment by victoriaatraft** [comment]
> @elipe17 @victoriaatraft i left comments on a couple of line changes that can be prioritized for merging, perhaps on another branch. defer to dev on how best to go about that.

Thanks @ADPennington! checked in with @elipe17 and will be tackling this, this afternoon and wrap up by tomorrow morning

**Comment by elipe17** [comment]
> @elipe17 @victoriaatraft i left comments on a couple of line changes that can be prioritized for merging, perhaps on another branch. defer to dev on how best to go about that.

@ADPennington, @victoriaatraft I pulled the changes into another branch: `merge-first-changes` and made that branch this branch's base branch. I tagged everyone on the new PR.

**Review by victoriaatraft — COMMENTED** [review]
Eric pulled the changes into another branch: merge-first-changes and made that branch this branch's base branch and tagged everyone on the new PR. -- ignore, tried to dismiss a request change below

**Comment by victoriaatraft** [comment]
@ADPennington @lfrohlich  I know it says change requested, I tried to resolve that but those changes were addressed by Eric :)

**Comment by ADPennington** [comment]
> @ADPennington @lfrohlich I know it says change requested, I tried to resolve that but those changes were addressed by Eric :)

thank you.  just one quick question on this. Any suggestions on an efficient way to test this? For context, all of the friendly names have changed with this PR which currently only appear in the error reports. I usually confirm these sorts of changes by submitting a test file that would yield the relevant error messages that have changed. I'm not sure if we have some test files relevant to this PR to help make this a little easier. @victoriaatraft @reitermb @elipe17

**Comment by elipe17** [comment]
> > @ADPennington @lfrohlich I know it says change requested, I tried to resolve that but those changes were addressed by Eric :)
> 
> thank you. just one quick question on this. Any suggestions on an efficient way to test this? For context, all of the friendly names have changed with this PR which currently only appear in the error reports. I usually confirm these sorts of changes by submitting a test file that would yield the relevant error messages that have changed. I'm not sure if we have some test files relevant to this PR to help make this a little easier. @victoriaatraft @reitermb @elipe17

@ADPennington there is not an easy way for us to do this at the moment. Best we can do is submit files that have lots of errors, e.g. `ADS.E2J.FTP1.TS06`, and do a manual verification. The only way we might be able to actually test this programmatically is if we built a file generator that allowed us to specify if we want to fuzz the values for the record types. Then we could generate files that have an error for every single field in each record type quite easily.

**Comment by ADPennington** [comment]
unit tests now failing on this @elipe17 @victoriaatraft

**Comment by elipe17** [comment]
> Details

Ah yes, these failures are result of the recent PRs being merged. @ADPennington, @victoriaatraft I will take a look shortly.

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py]:
```suggestion
                    "Hours of Excused Absences",
```

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py]: @elipe17 why this change? 

WPS_49

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py]: @victoriaatraft here you add "ethnicity/race:" but not for M2, M3, T2,  Tribal T2, Tribal T3, M5, T5, and Tribal T5. Please inform what the expected language should be. cc: @elipe17

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py]:
```suggestion
                    friendly_name="Hispanic or Latino",
```

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py]:
```suggestion
            friendly_name='SSP-MOE Recipients',
```

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py]:
```suggestion
            friendly_name="Total Amount of Cash Assistance",
```

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py]:
```suggestion
            friendly_name="Total Number of Approved Applications",
```

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py]:
```suggestion
            friendly_name="Total Amount of Cash Assistance",
```

File: tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t6.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t6.py]:
```suggestion
            friendly_name="Total Number of One-Parent Families",
```

File: tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t6.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t6.py]:
```suggestion
            friendly_name="Total Number of No-Parent Families",
```

File: tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t6.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t6.py]:
```suggestion
            friendly_name="Total Number of Applications",
```

File: tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t6.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t6.py]:
```suggestion
            friendly_name="Total Number of No-parent Families",
```

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m7.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m7.py:74]:
```suggestion
                    friendly_name="TDR Section Indicator",
```

File: tdrs-backend/tdpservice/settings/common.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/settings/common.py]: @elipe17 can the syntax that is commented out be removed?

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py]:
```suggestion
```

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py]:
```suggestion
```

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py]:
```suggestion
                        validators.isInLimits(1, 2),
```

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py]: please revert. @elipe17

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py]: @elipe17 please revert.

File: tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t2.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t2.py]:
```suggestion
                    friendly_name="Amount of Unearned Income: Social Security",
```

**Review by ADPennington — CHANGES_REQUESTED** [review]
@victoriaatraft @elipe17 changes requested

**Comment by elipe17** [comment]
@victoriaatraft and @ADPennington I resolved the suggestions/changes that were in my purview. I left a few open for Victoria/@reitermb to address.

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py
Comment (victoriaatraft) [reply tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py]: Thank you. I'll updated to remove "ethnicity/race" @ADPennington

**Comment by victoriaatraft** [comment]
> @elipe17 @ADPennington resolved open suggestion and made the last to Alex's open comment. We're removing "Race/Ethnicity" thank you

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py]: @victoriaatraft @elipe17 please confirm if this change is the correct one or the one located in [line 574](https://github.com/raft-tech/TANF-app/blob/880f13b5032f0e178ea45c1e9831279654434fcb/tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py#L574)

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py]: @elipe17 why this change?

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py]: @elipe17 same question

File: tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py
Comment (ADPennington) [inline tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py]: this field is not optional. @elipe17

**Review by ADPennington — CHANGES_REQUESTED** [review]
@elipe17 a few changes requested. Just want to ensure that we are only making friendly name changes here. when these changes are fixed. please merge. @cc: @victoriaatraft @robgendron

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py
Comment (elipe17) [reply tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py]: Looks like this got clobbered in a merge somehow.

File: tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py
Comment (elipe17) [reply tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py]: Looks like this got clobbered in a merge somehow.

**Review by ADPennington — APPROVED** [review]
🚀

**Pull request merged**
